### ISSUE TITLE
Allow 1.1 values for aria-haspopup in 1.0 pattern

### ISSUE THREAD

**Issue by carmacleod** [w3c/aria-practices#1025]
The [Combobox pattern](http://w3c.github.io/aria-practices/#combobox) says:
> In a combobox implementing the ARIA 1.1 pattern:
> - If the combobox popup has a role other than listbox, the element with role combobox has aria-haspopup set to a value that corresponds to the popup type. That is, aria-haspopup is set to grid, tree, or dialog.
>
> In a combobox implementing the ARIA 1.0 pattern:
> - the element with role combobox has a value for aria-haspopup of listbox. Note that elements with role combobox have an implicit aria-haspopup value of listbox.

This implies that a 1.0 combobox cannot have, for example, `aria-haspopup="dialog"`, but the spec does not restrict the value of aria-haspopup for 1.0 comboboxes.

**Comment by mcking65** [comment]
Closing because this is no longer necessary; we changed combobox for APG 1.2 so it has only 1 pattern for combobox.
